irohad/model/commands/transfer_asset.hpp

```diff
@@ -45,7 +46,7 @@ namespace iroha {
       /**
        * Amount of transferred asset
        */
-      std::string amount;
+      std::decimal::decimal64 amount;
```

Comment:
We are already fixing it. We will use our implementation of Amount, simply two ints.